staff/almu-itur/cv/index.html

```diff
@@ -0,0 +1,78 @@
+index <!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>CV ALMUDENA ITURGOYEN</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <header>
+            <h1>ALMUDENA ITURGOYEN</h1>
+            <h2>Aspiring developer</h2>
+            <h6>Email</h6><h6>Phone</h6>
```

Comment:
it is very strange to tag email and phone as headings (h6), may you want to list them or just include them in an address tag?

Reply:
Just some information, I am changing it for a p tag